crates/ide_assists/src/handlers/generate_function.rs

```diff
@@ -746,16 +746,18 @@ fn bar(worble: ()) ${0:-> ()} {
     fn add_function_with_impl_trait_arg() {
         check_assist(
             generate_function,
-            r"
+            r#"
+#[lang = "sized"] trait Sized {}
```

Comment:
why do we need this here?

Reply:
It would generate
```rust
fn bar(foo: impl Foo + ?Sized) ${0:-> ()} {
    todo!()
}
```
otherwise.

I didn't handle missing `Sized` lang item specially when printing sources, I though it is unlikely.

Reply:
> I guess `minicore` isn't wired up in the assist tests?

Ha, it is!